### DISCUSSION TITLE
Allow for temporary chats

### DISCUSSION THREAD

**Discussion by allo-** [open-webui/open-webui#6621] [General]
# Feature Request
Allow to create chats that are by default not recorded into the history.

Use-Case:
Open Webui can nicely be integrated into the Firefox AI sidebar, but using it like this clutters the chat list with many small chats like "summarize this text: (Firefox Selection)" that are later no longer relevant.

A simple option for temporary chats, like `?q=my+input&temp=true` (with `temp=true` meaning it is a temporary chat that should not be saved by default), for starting a chat that is not recorded would be very helpful.

Temporary chats can of course have a "save to my chats" button to keep them afterward.

**Comment by allo-** [comment]
Why was this converted into a discussion? It's meant as concrete feature request.

**Reply by tjbck** [reply]
We already have `/?temporary-chat=true`!

**Reply by allo-** [reply]
Oh, that works for me! I just didn't find it.

So maybe you'd like to include the Firefox howto in your documentation:

- Set `browser.ml.chat.enabled = true`
- Set `browser.ml.chat.hideLocalhost = false`
- Set `browser.ml.chat.provider = http://BASEURL?temporary-chat=true`
- Optionally set `browser.ml.chat.shortcuts.custom = true` for custom prompts.
- Enjoy Open-Webui in your Firefox sidebar!
